commands/help.ts

```diff
@@ -39,15 +45,31 @@ export class HelpCommand implements ICommand {
 				while (substitutionPoint = helpText.match(this.$injector.dynamicCallRegex)) {
 					this.$logger.trace(substitutionPoint);
 					var data = this.$injector.dynamicCall(substitutionPoint[0]).wait();
-
 					var pointStart = substitutionPoint.index;
-					var pointEnd = pointStart + substitutionPoint[0].length;
-					helpText = helpText.substr(0, pointStart) + data + helpText.substr(pointEnd);
+					if(typeof data === "boolean") {
```

Comment:
The code is correct, but see https://lodash.com/docs#isBoolean . I do not insist on changing it.